source/globalCommands.py

```diff
@@ -1540,68 +1540,68 @@ def script_reportAppModuleInfo(self,gesture):
 
 	def script_activateGeneralSettingsDialog(self, gesture):
 		wx.CallAfter(gui.mainFrame.onGeneralSettingsCommand, None)
-	# Translators: Input help mode message for go to general settings dialog command.
-	script_activateGeneralSettingsDialog.__doc__ = _("Shows the NVDA general settings dialog")
+	# Translators: Input help mode message for go to general settings command.
+	script_activateGeneralSettingsDialog.__doc__ = _("Shows the general settings in the NVDA settings dialog")

```

Comment:
Hmm, so you're saying leave it as it is? I'm happy to do that, but it is literally speaking not valid, as there is no general settings dialog any more.

How about something like "Shows NVDA's general settings"?